lib/dialogs.dart

```diff
@@ -10,6 +10,8 @@ import 'dart:html';
 import 'elements/elements.dart';
 import 'sharing/gists.dart';
 import 'sharing/mutable_gist.dart';
+import 'package:dart_pad/core/keys.dart';
+import 'package:dart_pad/dart_pad.dart';
```

Comment:
I'd like to just tackle showing keybindings for now; settings can come later, once there's general agreement about how they should be surfaced.